ERROR in only one instance of babel-polyfill is allowed  @ ./~/bootstrap-loader/loader.js 1:0-44

I upgrade <code>bootstrap-loader</code> according #13, and it work for node@0.12.0 and earlier, but I got another problem for <code>babel-polyfill</code> is loaded repeatedly.

the error message is in the following:
```
ERROR in only one instance of babel-polyfill is allowed  @ ./~/bootstrap-loader/loader.js 1:0-44
```
The root cause is I wrote my dev webpack config with es6, and I add <code>require('babel/register');</code> this line in my <code>webpack-dev-server.js</code>

Anyway, if re-write whole webpack config with es5 and it's work, so have any idea to solve it or did I miss something?

Thanks :)

@alexfedoseev, finally, I've time to test this issues, but some problem still remain
I use babel 5 and I downgrade nodejs from 4.x to 0.12.0, because my project required.

Then I remove node_modules and install again and run my webpack dev command, but got the following message: 
```
ERROR in Cannot find module 'babel-polyfill'
 @ ./~/bootstrap-loader/loader.js 1:0-44
```

There's a first problem: 
For the user of babel@5, I think  ```babel/polyfill``` is necessary instead of ```babel-polyfill```

However, I just install babel-polyfill@6.3.14 and run again, but got another issues
```
ERROR in only one instance of babel-polyfill is allowed
 @ ./~/bootstrap-loader/loader.js 1:0-44
```

It's the second problem
This problem is same as my first post in this thread, because I wrote my webpack with es6 and run following command to execute
```bash
node --harmony ./webpack/webpack-dev-server.js
```
so I need to add this ```require('babel/register');``` at first line in ```webpack-dev-server.js``` for es6 compatibility. But this will cause a duplicate of babel instance.  

Anyway, I've solved this problem by using ```babel-node``` to run instead of ```node --harmony```,  cause of this solution, I can remove ```require('babel/register');``` to avoid the duplicate of babel instance.

--- a/src/bootstrap.loader.js
+++ b/src/bootstrap.loader.js
@@ -16,11 +16,12 @@ if (semver.lt(process.version, '4.0.0')) {
       Make sure it's installed in your 'node_modules/' directory.
     `);
   }
-
-  if (isBabelLatest) {
-    require(babelLatest);
-  } else {
-    require(babelPrev);
+  if (!global._babelPolyfill) {
+    if (isBabelLatest) {
+      require(babelLatest);
+    } else {
+      require(babelPrev);
+    }
   }
 }
 

